--- a/src/komega/CMakeLists.txt
+++ b/src/komega/CMakeLists.txt
@@ -16,4 +16,3 @@
 if(MPI_FOUND)
   target_link_libraries(komega ${MPI_Fortran_LIBRARIES})
 endif(MPI_FOUND)
-install(TARGETS komega ARCHIVE DESTINATION lib LIBRARY DESTINATION lib RUNTIME DESTINATION bin)
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@
 endif(NOT CMAKE_BUILD_TYPE)
 message(STATUS "Build type: " ${CMAKE_BUILD_TYPE})
 
-option(BUILD_SHARED_LIBS "Build shared libraries" ON)
+option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
 option(ENABLE_MPI "Enable MPI Parallelization" ON)
 enable_language(C Fortran)
 
--- a/tool/CMakeLists.txt
+++ b/tool/CMakeLists.txt
@@ -12,20 +12,20 @@
 add_executable(cTPQ cTPQ.F90)
 target_link_libraries(fourier key2lower ${LAPACK_LIBRARIES})
 
-install(TARGETS fourier RUNTIME DESTINATION bin)
-install(TARGETS corplot RUNTIME DESTINATION bin)
+install(TARGETS fourier RUNTIME DESTINATION share/hphi/tool)
+install(TARGETS corplot RUNTIME DESTINATION share/hphi/tool)
 #
 # Scripts
 #
 configure_file(gen_frmsf.sh gen_frmsf.sh COPYONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/gen_frmsf.sh DESTINATION bin
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/gen_frmsf.sh DESTINATION share/hphi/tool
         PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 configure_file(wout2geom.sh wout2geom.sh COPYONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/wout2geom.sh DESTINATION bin
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/wout2geom.sh DESTINATION share/hphi/tool
         PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 configure_file(histgram.sh histgram.sh COPYONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/histgram.sh DESTINATION bin
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/histgram.sh DESTINATION share/hphi/tool
         PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 configure_file(AveSSrand.py AveSSrand.py COPYONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/AveSSrand.py DESTINATION bin
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/AveSSrand.py DESTINATION share/hphi/tool
         PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
